yt_dlp/extractor/twitch.py

```diff
@@ -314,6 +376,33 @@ def _extract_info_gql(info, item_id):
         if thumbnail:
             for p in ('width', 'height'):
                 thumbnail = thumbnail.replace('{%s}' % p, '0')
+
+        chapters = list()
```

Comment:
convention is to do
```suggestion
        chapters = []
```
Also, this entire for loop looks like it can be simplified into a list comprehension

Reply:
tried to change the loop to a list comprehension, is this how you meant?